egs/rm/s5/local/prepare_wsj_rm_lang.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+# Copyright 2017 Pegah Ghahremani
+
+# This script prepares a dictionary for sourc-target using source phone set, lexicon and dict and
```

Comment:
I planned to make it generic, but it is tough to make it generic. 
The name is wsj_rm, so it should not mislead the user!